src/storage/table_test.cc

```diff
@@ -380,7 +367,9 @@ TEST_P(TableTest, SchedGcHead) {
             ASSERT_FALSE(table->IsExpire(entry));
         }
     }
-    ASSERT_EQ(1, (int64_t)table->GetRecordCnt());
+    if (storageMode == ::openmldb::common::StorageMode::kMemory) {
+        ASSERT_EQ(1, (int64_t)table->GetRecordCnt());
```

Comment:
what's the behavior of `disk table`?

Reply:
`GetRecordCnt` will return 0. But maybe we do not expect it to be exactly 0 as it's an estimated number.